improve readmes (#93)
kinvolk/lokomotive-kubernetes: digital-ocean/container-linux/kubernetes/README.md

> Support for Digital Ocean isn't under active developing now, so there is not an updated documentation for it.

Reviewer: ```suggestion
Support for Digital Ocean isn't under active developing now, so there is no up-to-date documentation for it.
```
